c/src/ml-api-service-remote.c

```diff
@@ -576,22 +610,26 @@ ml_service_remote_create (ml_option_h option, ml_service_event_cb cb,
   remote_s->node_type = edge_info->node_type;
   remote_s->event_cb = cb;
   remote_s->user_data = user_data;
+  if (ML_ERROR_NONE == ml_option_get (option, "path", &value)) {
```

Comment:
may check `G_FILE_TEST_IS_DIR` here?

Reply:
Good, I changed to check `G_FILE_TEST_IS_DIR` here.